docs/user/jobflowremote.md

```diff
@@ -225,7 +221,11 @@ print(jc.get_jobs_doc(db_ids='214')[0].job.maker.input_set_generator.user_incar_
 2. `jf flow list` (list of flows in the DB)
 3. `jf job info jobid` (provides some info of job like workdir, error info if it failed)
 4. `jf flow delete -did db_id` (deletes flow from DB)
-5. `jf flow -h` or `jf job -h` for checking other options
+5. `jf job list -s STATE`, e.g. `jf job list -s FAILED`
+6. `jf job rerun -s STATE`, e.g. `jf job rerun -s FAILED`
+7. `jf job retry -s STATE`, e.g. `jf job retry -s REMOTE_ERROR`
+8. `jf flow info flow_id` (shows the jobs of a certain flow)
+9. `jf flow -h` or `jf job -h` for checking other options
```

Comment:
Maybe we also add a note of needing to restart runner if connection to remote cluster is broken?

and also to check `jf job rerun -f` (force rerun option?)

Reply:
ok, I'll do it